examples/java/direct/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+
+apply plugin: org.apache.beam.gradle.BeamModulePlugin
+// Disable default shadow jar closure and include all class files and resources.
+applyJavaNature(shadowClosure: {})
```

Comment:
It makes a lot of sense to actually place these builds underneath each runner instead of underneath examples.

Reply:
That might make more sense. However, we aggregating the example classes and resources here. Is it reasonable to depend on the examples modules from runner (sub)modules?

Reply:
Yes, its much cleaner as the owner of the testing is really the runner and not the examples module.